analysis/cpp/fast_em.cc

```diff
@@ -194,7 +199,10 @@ static void ExpectationMaximization(
 
     pij = new_pij;  // copy
 
-    log("max_dif: %e", em_iter, max_dif);
+    if (em_iter % 200 == 0) {
```

Comment:
I changed the output of fast_em.cc in the TensorFlow branch which Tom is reviewing... so these changes will probably conflict.  It matches the R output more closely now.

I think it's better to just revert these changes since they're not really related.